src/style.css

```
@@ -1,3 +1,74 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: inline-block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: block;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  padding-right: 20px;
+}
+
+.nav__item:last-child {
+  padding-right: 0;
+}
```

you should use `margin` to make an outer indent between siblings
```suggestion
.nav__item:not(:last-child) {
  margin-right: 20px;
}
```